pkgs/applications/misc/solaar-master/default.nix

```diff
@@ -0,0 +1,42 @@
+{fetchFromGitHub, stdenv, gtk3, python34Packages, gobjectIntrospection}:
+python34Packages.buildPythonApplication rec {
+  name = "solaar-${version}";
```

Comment:
the name part should be `solaar-unstable` and the version part should be the date, `YYYY-MM-DD`
https://nixos.org/nixpkgs/manual/#sec-package-naming

Reply:
Given the age of the current release it might be better to upgrade the current stable package to master rather maintaining two packages.

Reply:
thats what I thought as well
would we go and remove that stable package or alter it to point to the master branch?

Reply:
Just point it to master.